examples/multi_agent.py

```diff
@@ -65,10 +61,15 @@ def main(scenarios, sim_name, headless, num_episodes, seed, max_episode_steps=No
     parser = default_argument_parser("multi-agent-example")
     args = parser.parse_args()
 
+    if not args.scenarios:
+        args.scenarios = [
+            str(pathlib.Path(__file__).absolute().parents[1] / "scenarios" / "loop")
+        ]
+
+    build_scenario(args.scenarios)
```

Comment:
```suggestion
    if not args.scenarios:
        args.scenarios = [
            str(pathlib.Path(__file__).absolute().parents[1] / "scenarios" / "loop")
        ]

        build_scenario(args.scenarios)
```
I don't think its a good idea to rebuild the scenario if it was passed as an argument. I can see that the user can forget to build the scenario before running an example on it, but we have check for those cases down the stream.

Reply:
I desired the code to run seamlessly even if the user forgets to pass a scenario argument or forgets to build the scenario. Catching unbuilt scenario later down stream and throwing an error, kind of spoils the seamless experience. 

I have removed the `--clean` option from the `scl scenario build-all` command, which should reduce some of the things regenerated. 

Please do suggest, if you have other better code ideas.